src/main/scala/io/iohk/ethereum/blockchain/data/GenesisDataLoader.scala

```diff
@@ -83,30 +85,28 @@ class GenesisDataLoader(
     implicit val formats: Formats = DefaultFormats + ByteStringJsonSerializer
     for {
       genesisData <- Try(parse(genesisJson).extract[GenesisData])
-      _ <- loadGenesisData(genesisData)
+      consensusContractCode <- Try(Utils.loadContractCodeFromFile(new File(ouroborosConfig.consensusContractFilepath)))
```

Comment:
Minor detail: Maybe we should rename `consensusContractCode` to `consensusInitContractCode`?